test/ui-test/testSuites/global_shared/steps/startupSteps.py

```diff
@@ -0,0 +1,81 @@
+"""It defines starting-up or driving-the-app-into-an-idle-state static methods outside bdd context that can be reused in different `hooks` as well as in specific bdd steps files."""
+
+from utils.FileManager import *
+from common.Common import *
+
+from screens.StatusWelcomeScreen import StatusWelcomeScreen
+from screens.StatusMainScreen import StatusMainScreen
+from screens.StatusChatScreen import StatusChatScreen
+
+# Project settings properties:
+_status_desktop_app_name = "nim_status_client"
+_status_data_folder_path = "../../../../../Status/data"
+_status_fixtures_folder_path = "../../../fixtures/"
+_status_shared_images_path = "../shared/searchImages/"
+_status_qt_path = "../../../../../Status/qt"
+_app_closure_timeout = 2 #[seconds]
+
+# Test context properties names:
+_aut_name = "aut_name"
+_status_data_folder = "status_data_folder_path"
+_fixtures_root = "fixtures_root"
+_search_images = "search_images"
+_scenario_name = "scenario_name"
+
+# Test static objects creation:
+_welcome_screen = StatusWelcomeScreen()
+_main_screen = StatusMainScreen()
+_chat_screen = StatusChatScreen()
+    
+def context_init(context):
+    erase_directory(_status_qt_path)
+    context.userData = {}
+    context.userData[_aut_name] = _status_desktop_app_name
+    context.userData[_status_data_folder] = _status_data_folder_path
+    context.userData[_fixtures_root] = os.path.join(os.path.dirname(__file__), _status_fixtures_folder_path)
```

Comment:
This one is overwritten on line 52, so maybe this line is useless

Reply:
That's true. I will remove it in my next pr (https://github.com/status-im/status-desktop/pull/7987) that also modifies this file.